src/coins.h

```diff
@@ -528,7 +572,13 @@ class CCoinsViewCache : public CCoinsViewBacked
 private:
     CCoinsMap::iterator FetchCoins(const uint256 &txid);
     CCoinsMap::const_iterator FetchCoins(const uint256 &txid) const;
+    CSidechainsMap::const_iterator FetchSidechains(const uint256& scId) const;
+    bool hasScCreationOutput(const CTransaction& tx, const uint256& scId);
+    static int getInitScCoinsMaturity();
+    int getScCoinsMaturity();
```

Comment:
These methods probably belong to sidechain.h or transaction.h files.
With @alsala  we decided to postpone these changes since it has proven strangely difficult to move them out, and since it could be smarted to reshuffle code following merging with certificate related code changes